doc/model/axes/axis-cartesian-discrete.xml

```diff
@@ -32,6 +32,25 @@
             For additional information, please see <c:link to="pvc.options.axes.CartesianAxis" />.
         </c:documentation>
 
+        <c:property name="labelRotationDirection" 
+                    type="pvc.options.varia.AxisLabelRotationDirection"
+                    default="'clockwise'"
+                    category="Non-Hierarchical > Layout">
+            <c:documentation>
+                Controls in which direction labels rotate.
```

Comment:
Please include the suffix: "Controls in which direction labels rotate (applies to discrete axes)."